### DISCUSSION TITLE
How to stop writing "new job recieved..." into log?

### DISCUSSION THREAD

**Discussion by Gl4d1u5** [Lolliedieb/lolMiner-releases#205] [Q&A]
Is there a way to stop writing this line into a log?
I couldn't find a way in the documentation.

**Comment by yumyumfarted** [comment]
Short answer: 
You can't. 

Long answer: 
Logging will write every output to a file, so you wouldn't be able to change it. You theoretically could write a script that reads the output of the miner, and writes everything to a new log file without the line string you mentioned. This requires some work though, I'm not too familiar with JS or other scripting languages.
